docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -6,17 +6,24 @@ mapped_pages:
 
 # filestream input [filebeat-input-filestream]
 
+::::{important}
+Filestream, with default settings will only start ingesting files **when
+their size is >= 1024 bytes**. To change this behaviour you can
+configure a different length for the fingerprint by setting
+[`prospector.scanner.fingerprint.length`](#filebeat-input-filestream-scan-fingerprint).
+::::
 
 Use the `filestream` input to read lines from active log files. It is the new, improved alternative to the `log` input. It comes with various improvements to the existing input:
```

Comment:
I would probably use an unordered list here instead since the list items aren't sequential.

Reply:
That makes a lot of sense! Thanks for the suggestion :smile: